app/views/hackdays/_project_list.html.erb

```diff
@@ -0,0 +1,36 @@
+<% if hackday.votable %>
```

Comment:
There's 2 parts which are different- contents of "project-votes" and whether the list is sorted.  sorting needs to happen at the beginning and I don't know how to store a "form_for" form, so wouldn't that require 2 ifs?